### PR TITLE
Cronify sending staffing request inquiries

### PR DESCRIPTION
**Pull request by kkamalov**

### CODE REVIEW

File: orchestra/admin.py [hunk 1]
```diff
@@ -205,7 +205,7 @@ class CommunicationPreferenceAdmin(admin.ModelAdmin):
 @admin.register(StaffingRequestInquiry)
 class StaffingRequestInquiryAdmin(RelatedFieldAdmin):
     list_display = (
-        'id', 'project_description', 'communication_preference__worker'
```
Comment (joshblum) [inline orchestra/admin.py]: we should figure out how to leave this in

File: orchestra/cron.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import kronos
+from orchestra.communication.staffing import send_staffing_requests
+
+
+@kronos.register('*/5 * * * *')
```
Comment (joshblum) [inline orchestra/cron.py]: put a comment for how frequent this is in minutes

**Comment by joshblum** [comment]
to verify: there is a way to run the task as a management command?

**Comment by joshblum** [comment]
run it every minute instead: http://www.cyberciti.biz/faq/run-crontab-job-every-minute-on-linux-unix-bsd/

**Comment by joshblum** [comment]
TODO: update bot docs with kronos setup

File: example_project/example_project/orchestra_settings.py [hunk 3]
```diff
@@ -161,6 +161,8 @@ def setup_orchestra(settings_module_name):
 
     # Notification-specific email for message bundling and searching
     settings.ORCHESTRA_NOTIFICATIONS_FROM_EMAIL = settings.DEFAULT_FROM_EMAIL
+    # Used to test email sending in development/staging environments
```
Comment (marcua) [inline example_project/example_project/orchestra_settings.py:164]: newline

**Pull request closed without merging**